### ISSUE TITLE
`--version` flag reports unexpected version number

### ISSUE THREAD

**Issue by toastal** [parcel-bundler/lightningcss#322]
Following up with [my comment](https://github.com/parcel-bundler/lightningcss/pull/284#issuecomment-1249001999)...

```console
$ lightningcss --version
1.0.0-alpha.34
```

What I would have expected is it to match the tag

```console
$ lightningcss --version
1.16.0
```

This is blocking using the `testers.testVersion` for Nixpkgs @ https://github.com/NixOS/nixpkgs/pull/191453 because the version does not match the release tag version.

**Comment by devongovett** [comment]
Yeah it's the version of the Rust crate. I have been too scared to tag a v1 😂

**Comment by fvsch** [comment]
Ran into this today, and it is very confusing. 😅

(My use case is limited, I’m just used to call a CLI tool with `--version` as a smoke test, to check that it’s installed and not immediately crashing.)
